### ISSUE TITLE
Feature request: Minimum Call Cost

### ISSUE THREAD

**Issue by DigiDaz** [cgrates/cgrates#991]
It would be great if we could have this as a field somewhere.

**Comment by ghenry** [comment]
I've been trying to work a solution using minimum call duration like so (
min-charge /  ( rate / 60)) = duration for GroupIntervalStart, hence this
ticket.

**Comment by DigiDaz** [comment]
Yep, that would work but its a PITA to implement.

I think I have mentioned this to Dan in the past but maybe then I was the lonely dude asking for it. 

Maybe if more are interested the cgrates guys will implement it.

**Comment by ghenry** [comment]
Yeah, it's just a bit of maths though when you generate the rate, but it's
made me think of switching to a minimum call duration instead or connection
charge.

It is a very common billing model in the UK ITSP market though. I've seen
minimum durations used with a connection fee / setup fee too, which is
expensive. With bundles that doesn't matter so it would encourage adoption.

On most > 1 minute calls it doesn't matter anyway.

Gavin.

--
Kind Regards,

Gavin Henry.
Managing Director.

T +44 (0) 330 44 50 000
D +44 (0) 330 44 55 007
M +44 (0) 7930 323266
F +44 (0) 1224 824887
E ghenry@suretec.co.uk

Open Source. Open Solutions(tm).

http://www.suretecsystems.com/

Winner of the Best Business ITSP (Medium Enterprise) 2016!
http://www.surevoip.co.uk/2016-best-provider

Suretec Systems is a limited company registered in Scotland. Registered
number: SC258005. Registered office: 24 Cormack Park, Rothienorman,
Inverurie, Aberdeenshire, AB51 8GL.

Subject to disclaimer at http://www.suretecgroup.com/disclaimer.html

Do you know we have our own VoIP provider called SureVoIP? See
http://www.surevoip.co.uk

OpenPGP (GPG/PGP) Public Key: 0x8CFBA8E6 - Import from hkp://
pool.subkeys.pgp.net
or http://www.suretecgroup.com/0x8CFBA8E6.gpg

**Comment by danbogos** [comment]
Hey guys,

We will implement it for sure. It is just a matter of time whether we do it in existing version of RatingPlans or in the next one.

DanB

**Comment by ennala** [comment]
We require this feature too.

Venkatesh Ennala.

**Comment by danbogos** [comment]
Should be a part of the new rates which we plan for the future.

DanB
